runtime/compiler/env/j9methodServer.cpp

```diff
@@ -166,8 +166,22 @@ TR_ResolvedJ9JITaaSServerMethod::classOfStatic(I_32 cpIndex, bool returnClassFor
    // if method is unresolved, return right away
    if (cpIndex < 0)
       return nullptr;
+
+   auto &classOfStaticCache = getJ9ClassInfo(_fe->_compInfoPT, _ramClass)._classOfStaticCache;
```

Comment:
classOfStaticCache is a pointer. Do you need the `&` ?

Reply:
Yes, because if classOfStaticCache is NULL, we need to initialize it, which means we need to change the value of the pointer in ClassInfo.